taskt/Core/Automation/Commands/Folder Operation Commands/DeleteFolderCommand.cs

```diff
@@ -1,59 +1,59 @@
-    [Attributes.ClassAttributes.UsesDescription("Use this command to delete a folder from a specific location.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements '' to achieve automation.")]
+    [Group("Folder Operation Commands")]
+    [Description("This command deletes a folder from a specified location.")]
     public class DeleteFolderCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the path to the source folder")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the folder.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\myfolder or [vTextFolderPath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Folder Path")]
+        [InputSpecification("Enter or Select the path to the folder.")]
+        [SampleUsage(@"C:\temp\myfolder || {ProjectPath}\myfolder  || {vTextFolderPath}")]
+        [Remarks("{ProjectPath} is the directory path of the current project.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)] 
         public string v_SourceFolderPath { get; set; }
 
         public DeleteFolderCommand()
         {
-            this.CommandName = "DeleteFolderCommand";
-            this.SelectionName = "Delete Folder";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "DeleteFolderCommand";
+            SelectionName = "Delete Folder";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
 
         public override void RunCommand(object sender)
         {
-
             //apply variable logic
             var sourceFolder = v_SourceFolderPath.ConvertToUserVariable(sender);
 
             //delete folder
-            System.IO.Directory.Delete(sourceFolder, true);
-
+            Directory.Delete(sourceFolder, true);
         }
+
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_SourceFolderPath", this, editor));
-
             return RenderedControls;
         }
+
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [delete " + v_SourceFolderPath + "']";
+            return base.GetDisplayValue() + $" [Folder Path '{v_SourceFolderPath}']";
```

Comment:
Please match the display value with DeleteFileCommand. So it would be [Delete '{v_SourceFilePath}'].

Reply:
You can change the one in DeleteFileCommand too. They just have to match

Reply:
changed to **[Delete '{v_SourceFolderPath}']**